### PR TITLE
Add scripts responsible for the ordering of repositories by aliases

### PR DESCRIPTION
**Pull request by Aeliot-Tm**
1. Add script responsible for the ordering of repositories by aliases
2. Add script which check if repositories is ordered by aliases
3. Config testing by github action
4. Add docker for the local development.

### CODE REVIEW

**Comment by Aeliot-Tm** [comment]
It should be merged after PR: https://github.com/phar-io/phar.io/pull/159 case it contains commit from it with adding of an alias

**Comment by Aeliot-Tm** [comment]
Hi @theseer, 

Could you have a look?

**Comment by Aeliot-Tm** [comment]
And another one PR fixes ordering of repositories https://github.com/phar-io/phar.io/pull/161

**Comment by theseer** [comment]
Can we add those scripts without all the docker crap?

File: data/repositories.xml
Comment (theseer) [inline data/repositories.xml:1]: This is unrelated and should not be included in this.

**Comment by Aeliot-Tm** [comment]
@theseer
Yep, I can extract it to another PR

File: data/repositories.xml
Comment (Aeliot-Tm) [reply data/repositories.xml:1]: Done

**Comment by theseer** [comment]
Much better :-)

I should probably add a GH action to auto publish the `repository.xml` file at some point. Right now, that's a  semi automated process..

**Comment by theseer** [comment]
Maybe you feel like tweaking this a little.
